src/prompt/branch_helpers.go

```diff
@@ -19,6 +22,9 @@ type askForBranchesOptions struct {
 
 func askForBranch(opts askForBranchOptions) string {
 	result := ""
+	if runtime.GOOS == "windows" {
+		surveyCore.SelectFocusIcon = ">"
+	}
```

Comment:
Should these updates be moved to some sort of init block? That way, all prompts get these update